### PR TITLE
LRDOCS-7778 Search Results

### PR DESCRIPTION
**Pull request by sez11a**
Did some significant wordsmithing and reorganizing. @rbohl if you don't like the changes, please let me know and I'll re-submit to you.

### CODE REVIEW

**Comment by rbohl** [comment]
Loos good to me :+1:

**Comment by jrhoun** [comment]
Just started reviewing.

File: docs/dxp/7.x/en/using-search/search-pages-and-widgets/search-results/filtering-search-results.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+![Apply a custom filter to weed out certain search results.](./filtering-search-results/images/03.png)
+
+There are many use cases satisfied by incorporating Custom Filters into your search page. 
+
+For now, consider these basic usages:
+
+| Use Case | `Configuration` _(Value)_ |
+| -----------| ----------- |
+| **User Title Filter:** End Users directly manipulate the filter key to refine results to the one they were looking for, using the Title |`Filter Field` _(title\_en\_US)_ <br /> `Filter Value` _(podcast)_ <br /> `Filter Query Type` _(Match)_ <br /> `Occur Type` _(must_not)_|
+| **Hard-Coded Filtering:** An administrative User configures the Search Page with filters that are invisible and unchangeable for end Users | ***Add to the above configuration*** <br /> `Invisible` _(True--checked)_ <br /> `Immutable` _(True--checked)_|
```
Comment (jrhoun) [inline docs/dxp/7.x/en/using-search/search-pages-and-widgets/search-results/filtering-search-results.md:14]: I don't understand these use cases - but I'm not sure they're meant to be understood by a non-search specialist - so I won't edit these at all. But I wanted to note that I read/re-read the use cases several times on different days and still feel my grasp is tenuous. I _think_ it's saying:

Case 1: You want your end users to be able to selectively enable/disable custom filters that you define in the UI to filter results. You enable this use case by configuring the following values...

Case 2: You want to filter results automatically and invisibly to search users. Therefore make sure you enable these configurations...

How close am I?

Comment (rbohl) [reply docs/dxp/7.x/en/using-search/search-pages-and-widgets/search-results/filtering-search-results.md:14]: Yeah you got it. This very brief use case information is just a stopgap until I write use case articles. The Custom Filter was designed to let people do things they used to have to write code to accomplish. It desperately requires use case docs to make it more useful. I have a ticket for this work in my queue, with use cases I have gathered from community slack and our internal slack's help-search channel.

Just need to clear some :timer_clock:

File: docs/dxp/7.x/en/using-search/search-pages-and-widgets/search-results/search-results.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+Results are filtered using *facets*. Most users have encountered similar filtering capabilities in other applications, particularly during commerce activities. Users enter a search term, are presented with a list of results, and selecting search facets refines the results. You can think of a facet as a bucket of results with a shared characteristics.
+
+![The Type facet is one of the facets provided out of the box.](./search-results/images/02.png)
+
+Facets are configurable. Read about [configuring facets](../search_facets.rst) to learn more.
+
+## Search Results Relevance
+
+The search engine contains the brains to order results by *relevance*. Relevance is a score calculated by the search engine. The score is calculated by algorithms provided by the [search engine](https://www.elastic.co/guide/en/elasticsearch/reference/current/relevance-intro.html#relevance-intro).
```
Comment (jrhoun) [inline docs/dxp/7.x/en/using-search/search-pages-and-widgets/search-results/search-results.md:29]: This link to elasticsearch material is broken, I tried to look for a replacement, but couldn't find one that I was confident would be suitable replacement. I'll remove this link for now and add a comment for future link TODO.

Comment (rbohl) [reply docs/dxp/7.x/en/using-search/search-pages-and-widgets/search-results/search-results.md:29]: Okay, if you're making edits and feel like dropping the below link in, that works too:

https://www.elastic.co/guide/en/elasticsearch/guide/master/relevance-intro.html#relevance-intro

**Comment by jrhoun** [comment]
Just pushed this up with some minor wordsmithing / formatting changes. Also added cards for the search pages and widgets section. See:

https://github.com/liferay/liferay-learn/commit/6a82607dee1a728b213585601aca8a3c9681d64c
https://github.com/liferay/liferay-learn/commit/0f52a29aa65047b7892f4c52204476529e1d826a
https://github.com/liferay/liferay-learn/commit/c8526e71c95ccc7534243099750d4e79083027ac

**Pull request closed without merging**